packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/lua
+
+require "ubus"
+local json = require 'luci.json'
+local ft = require('firstbootwizard.functools')
+local iwinfo = require("iwinfo")
+local wireless = require("lime.wireless")
+local fs = require("nixio.fs")
+local uci = require("uci")
+local nixio = require "nixio"
+
+local conn = ubus.connect()
+if not conn then
+    error("Failed to connect to ubus")
+end
+
+local function execute(cmd)
+    local f = assert(io.popen(cmd, 'r'))
+    local s = assert(f:read('*a'))
+    f:close()
+    return s
+end
+
+local function eui64(mac)
```

Comment:
A non issue really... but it would ve very interesting to get this one out into a library, because it is a very general funcitonality that can be used in many spaces